### PR TITLE
Feature/most used tags

### PR DESCRIPTION
**Pull request by alexandre-snr**
This PR adds the task update-most-used-tags which takes the most used tags out of the last tagging report and stores them in the SQL Db.

### CODE REVIEW

File: server/taskUpdateMostUsedTags.go [hunk 1]
```diff
@@ -0,0 +1,110 @@
...
+	}
+
+	err = updateMostUsedTagsForAccount(ctx, accountId)
+
+	if err != nil {
+		logger.Info("Task 'update-most-used-tags' done.", map[string]interface{}{
+			"args": args,
+		})
+	}
+	return nil
```
Comment (simonmeyerrr) [inline server/taskUpdateMostUsedTags.go]: should return err

File: tagging/routes/routeGetMostUsedTags.go [hunk 2]
```diff
@@ -0,0 +1,36 @@
+package routes
```
Comment (simonmeyerrr) [inline tagging/routes/routeGetMostUsedTags.go:15]: missing headers

File: tagging/routes/routes.go [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+
+// mostUsedTagsQueryArgs allows to get required queryArgs params
+var mostUsedTagsQueryArgs = []routes.QueryArg{}
+
+func init() {
+	routes.MethodMuxer{
+		http.MethodGet: routes.H(routeGetMostUsedTags).With(
+			db.RequestTransaction{db.Db},
+			users.RequireAuthenticatedUser{users.ViewerAsParent},
+			routes.QueryArgs(mostUsedTagsQueryArgs),
```
Comment (simonmeyerrr) [inline tagging/routes/routes.go]: don't need to set query args if you don't use them

**Review by simonmeyerrr — CHANGES_REQUESTED** [review]

**Review by jklakosz — COMMENTED** [review]
Seems globally good for me, not tested by my side.

**Review by simonmeyerrr — APPROVED** [review]

**Pull request merged**